demisto_sdk/commands/pre_commit/hooks/mypy.py

```diff
@@ -1,6 +1,14 @@
+from copy import deepcopy
+
 from demisto_sdk.commands.pre_commit.hooks.hook import Hook
 
 
 class MypyHook(Hook):
-    def prepare_hook(self, python_version: str, **kwargs):
-        self.hook["args"][-1] = f"--python-version={python_version}"
+    def prepare_hook(self, python_version_to_files: dict, **kwargs):
+        for python_version in python_version_to_files.keys():
+            hook = {"name": f"mypy-py{python_version}"} | deepcopy(self.base_hook)
+            hook["args"][-1] = f"--python-version={python_version}"
```

Comment:
let's remove the python version argument of the mypy, and use `hook["args"].append()` instead